composer.json

```diff
@@ -37,44 +36,46 @@
             "module": "DoctrineORMModule"
         }
     },
-    "require":           {
-        "php":                               "^7.1",
-        "doctrine/doctrine-module":          "^2.1.3",
-        "doctrine/orm":                      "^2.6.3",
-        "doctrine/dbal":                     "^2.6.0",
-        "symfony/console":                   "^3.3 || ^4.0 || ^5.0",
-        "zendframework/zend-stdlib":         "^3.2.1",
-        "zendframework/zend-hydrator":       "^2.3",
-        "zendframework/zend-mvc":            "^3.1",
-        "zendframework/zend-servicemanager": "^3.3"
+    "require": {
+        "php": "^7.2",
+        "doctrine/doctrine-module": "^3.0",
+        "doctrine/orm": "^2.7",
+        "doctrine/dbal": "^2.10",
+        "symfony/console": "^3.3 || ^4.0 || ^5.0",
+        "laminas/laminas-stdlib": "^3.2.1",
+        "laminas/laminas-mvc": "^3.1",
+        "laminas/laminas-servicemanager": "^3.3",
+        "laminas/laminas-dependency-plugin": "^1.0",
+        "laminas/laminas-code": "3.4.1",
+        "ocramius/proxy-manager": "^2.0"
     },
-    "require-dev":       {
-        "phpunit/phpunit":                    "^7.0.3",
-        "squizlabs/php_codesniffer":          "^2.7",
-        "doctrine/data-fixtures":             "^1.2.1",
-        "doctrine/migrations":                "^1.5 || ^2.0",
-        "zendframework/zend-console":         "^2.6",
-        "zendframework/zend-developer-tools": "^1.1",
-        "zendframework/zend-i18n":            "^2.7.3",
-        "zendframework/zend-log":             "^2.9",
-        "zendframework/zend-modulemanager":   "^2.7.2",
-        "zendframework/zend-mvc-console":     "^1.2",
-        "zendframework/zend-serializer":      "^2.8"
+    "require-dev": {
+        "phpunit/phpunit": "^7.5.20",
```

Comment:
Maybe we can use only ^8.5 ?

Reply:
8.5 requires PHP 7.3 and popular opinion is this should work @ PHP 7.2

I'd at least like a release which supports 7.2 then jump to 7.4, but the composer.json that's in place in this PR now allows for 7.2 to 7.4

Reply:
Correction phpunit 8.5 uses PHP 7.2.